src/Apps/Auctions/Components/MyBids/MyBids.tsx

```diff
@@ -129,38 +130,43 @@ export const MyBidsFragmentContainer = createFragmentContainer(MyBids, {
   `,
 })
 
-export const MyBidsQueryRenderer: React.FC = () => {
+interface MyBidsQueryRendererProps extends BoxProps {}
+export const MyBidsQueryRenderer: React.FC<MyBidsQueryRendererProps> = ({
+  mb = 0,
+}) => {
   const { relayEnvironment, user } = useSystemContext()
 
   if (!user) {
     return null
   }
 
   return (
-    <SystemQueryRenderer<MyBidsQuery>
-      lazyLoad
-      environment={relayEnvironment}
-      query={graphql`
-        query MyBidsQuery {
-          me {
-            ...MyBids_me
+    <Box mb={mb}>
```

Comment:
Since we're extending `BoxProps` in the interface, might as well expand things after line 135 and keep things truthful with:
```tsx
mb = 0, 
...boxProps
```
And then fully spread that in here:
```tsx
<Box mb={mb} {...boxProps}>
```